### ISSUE TITLE
ForkingPickler: Can't pickle local object 

### ISSUE THREAD

**Issue by hexaflexa** [eloialonso/iris#4]
I am trying to execute the example training run `python src/main.py env.train.id=BreakoutNoFrameskip-v4 common.device=cuda:0 wandb.mode=online`, but I am getting a `Can't pickle local object` error.  A partial backtrace is shown here:
  
```
Exception has occurred: AttributeError
  (note: full exception trace is shown but execution is paused at: _run_module_as_main)

Can't pickle local object 'Trainer.__init__.<locals>.create_env.<locals>.<lambda>'
  File "[PATH]\Lib\multiprocessing\reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)
  File "[PATH]\Lib\multiprocessing\popen_spawn_win32.py", line 93, in __init__
    reduction.dump(process_obj, to_child)
  File "[PATH]\Lib\multiprocessing\context.py", line 327, in _Popen
    return Popen(process_obj)
  File "[PATH]\Lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "[PATH]\Lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "src\envs\multi_process_env.py", line 61, in __init__
    p.start()
  File "src\trainer.py", line 66, in create_env
    return MultiProcessEnv(env_fn, num_envs, should_wait_num_envs_ratio=1.0) if num_envs > 1 else SingleProcessEnv(env_fn)
  File "src\trainer.py", line 74, in __init__
    test_env = create_env(cfg.env.test, cfg.collection.test.num_envs)
  File "src\main.py", line 9, in main
    trainer = Trainer(cfg)
```

Any recommendation on workarounds?

**Comment by vmicheli** [comment]
Hey, a simple workaround is to use a `SingleProcessEnv` instead of a `MultiProcessEnv` for the evaluation. You can do that by appending `collection.test.num_envs=1` to the training command.

Could you give me more details about your OS and Python version so that we can look into what is causing the issue?

**Comment by eloialonso** [comment]
Hey, the issue seems to be related to Windows. We included your suggestion in commit 03290c820e623d8b184ce53a082cccd05d3f08f4, thanks!
